docker/local/polygon-edge.sh

```diff
@@ -46,7 +46,8 @@ case "$1" in
                   rm -f /data/genesis.json
 
                   createGenesisConfig "$2" "$secrets" \
-                  --ibft-validators-prefix-path data-
+                    --ibft-validators-path /data \
+                    --ibft-validators-prefix-path data-
```

Comment:
WDYT about removing `ibft-validators-path` and `ibft-validators-prefix-path` flags from the `genesis` command and using `validators-path` and `validators-prefix` flags for both IBFT and Polybft? We can do that through a separate PR, if you find it possible.

Reply:
I'll do that within this PR

Reply:
Done